metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -1917,10 +1923,12 @@ def _query_published_datasource_for_project_luid(self, ds_luid: str) -> None:
                 self.datasource_project_map[ds_result.id] = ds_result.project_id
         except Exception as e:
             self.report.num_get_datasource_query_failures += 1
-            logger.warning(
-                f"Failed to get datasource project_luid for {ds_luid} due to error {e}"
+            self.report.warning(
+                title="Unexpected Query Error",
+                message="Failed to get datasource project_luid",
```

Comment:
project_luid is probably too technical for a user-facing error message
```suggestion
                message="Failed to get datasource details",
```